src/Athena/Content/css/site.css

```diff
@@ -78,16 +116,94 @@ main {
     background-color: #1a237e;
 }
 
+.btn:focus {
+    background-color: #1a237e;
+}
+
 .progress{
     height:2em;
     background-color:darkgrey;
 }
 
 .determinate {
     color: white;
-    background-color: #1a237e;
+    background-color: #0d47a1;
     display: inline-block;
     position: absolute;
     width: 70%;
     left: 0;
+}
+
+.cal-card-content{
+    background-color:lightgray;
+}
+
+user-tabs {
```

Comment:
This isn't a valid element name. Did you mean to ref the `user-tabs` id (`#user-tabs`) or class (`.user-tabs`)?

Reply:
period added